test_files/basic.untyped/basic.untyped.tsickle.ts

```diff
@@ -17,9 +17,9 @@ constructor(private ctorArg: string) {
 }
 
 function Foo_tsickle_Closure_declarations() {
-/** @type {?} */
+/** @type {?|undefined} */
```

Comment:
is this an intentional/good change? the original field was just `string`, this seems to slip in due to being untyped?